loopout.go

```diff
@@ -635,12 +709,75 @@ func (s *loopOutSwap) payInvoiceAsync(ctx context.Context,
 	}
 
 	// Lookup state of the swap payment.
-	paymentStateCtx, cancel := context.WithCancel(ctx)
+	payCtx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	payStatusChan, payErrChan, err := s.lnd.Router.SendPayment(
-		paymentStateCtx, req,
+	start := time.Now()
+	paymentStatus, attempts, err := s.sendPaymentWithRetry(
+		payCtx, hash, &req, maxRetries, routingPlugin, pluginType,
```

Comment:
I don't think we need to pass `maxRetries` if it's a config value within the scope of `loopOutSwap`?